lib/consumer_supervisor.ex

```diff
@@ -30,6 +30,8 @@ defmodule ConsumerSupervisor do
         use ConsumerSupervisor
```

Comment:
Actually I think this line can go as well, since we're not using the module-based supervisor. Correct?